examples/mock/mockoon-bb-information-mediator.json

```diff
@@ -0,0 +1,151 @@
+{
+  "uuid": "a11d79ca-aee0-468b-92c0-e52293cf2b6f",
+  "lastMigration": 24,
+  "name": "Mockoon bb information mediator",
+  "endpointPrefix": "",
+  "latency": 0,
+  "port": 3004,
```

Comment:
Just a guess. Shouldn't the port be set to 3003 ?

Reply:
This is just a port to test the API locally, e.g. with the Postman app, so it does not matter which port it will be :)